internal/consensus/common_test.go

```diff
@@ -41,7 +41,7 @@ const (
 	testSubscriber = "test-client"
 
 	// genesis, chain_id, priv_val
-	ensureTimeout = time.Millisecond * 200
+	ensureTimeout = time.Second
```

Comment:
This change will alter the timeouts in all of the `state_test.go` tests. I'm not a big fan of how these tests are implemented at the moment, and have [a branch](https://github.com/tendermint/tendermint/tree/wb/refactor-ensure) to refactor them a bit more so that this timeout can be adjusted for the different tests/calls, but for now this parameter is global :(. Is there a way we can adjust this in just some places? otherwise, this will slow down the consensus tests a lot.

Reply:
I will take a look. Without this and the other changes to the other timing parameters a lot of tests are failing. On my machine `make test` passes with these but i see that there are some failures in CI though. I was hoping to scale down all timeouts (and hopefully bring this back to 200ms).

Reply:
reverted this change